app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,23 @@
+class MoviesController < ApplicationController
+  def index
+    @movie = Movie.first
+    @index ||= Supports::Movie.new(@movie)
+  end
+
+  def list_movie
```

Comment:
list_movie với index thì khác gì nhau hả em?
check lại restfull nhé

Reply:
index là em đưa toàn bộ phim ra trang index, còn list movie của e là danh mục trên header và nó sẽ đưa phim ra theo từng thể loại chẳng hạn như phim bộ, hay là phim lẻ ạ

Reply:
Nếu list header thì nên đưa vào application controller và dùng before action